File: test_cases/python/gpt/11.py
def run_tests(solution_module):
    f = solution_module.minWindow
    assert f('ADOBECODEBANC', 'ABC') == 'BANC'
    assert f('a', 'a') == 'a'
    assert f('a', 'aa') == ''
    assert f('aa', 'aa') == 'aa'
    assert f('ab', 'b') == 'b'

File: test_cases/c/gpt/11.c
#include <assert.h>
#include <string.h>

char* minWindow(char* s, char* t);

int main(void) {
    assert(strcmp(minWindow("ADOBECODEBANC", "ABC"), "BANC") == 0);

    assert(strcmp(minWindow("a", "a"), "a") == 0);

    assert(strcmp(minWindow("a", "aa"), "") == 0);

    assert(strcmp(minWindow("AAABBC", "AABC"), "AABBC") == 0);

    assert(strcmp(minWindow("aAaBbBcC", "ABC"), "AaBbBc") == 0);

    return 0;
}